Documentation/project-docs/dogfooding.md

```diff
@@ -96,7 +96,8 @@ runtime.
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <RuntimeFrameworkVersion>3.0.0-preview-27225-1</RuntimeFrameworkVersion> <!-- modify build in this line -->
+     <!-- modify version in this line with one reported by `dotnet --info` under ".NET Core runtimes installed" -> Microsoft.NETCore.App -->
```

Comment:
Nit: Extra space at front. (I have noticed the suggestions insert extra spaces sometimes...)